.github/workflows/compile.yml

```diff
@@ -25,4 +25,4 @@ jobs:
     - name: Setup Android build environment
       uses: android-actions/setup-android@v2.0.2
     - name: Local install
-      run: ./gradlew clean uploadArchives -PSNAPSHOT=true -PLOCAL=true
+      run: ./gradlew clean publishToMavenLocal -PSNAPSHOT=true -PLOCAL=true
```

Comment:
-PLOCAL=true is not needed according to your README change?

Also, what commands are needed for publishing a snapshot and a release with these changes?

Reply:
Here is the reference docs for maven publishin: https://docs.gradle.org/current/userguide/publishing_maven.html

TLDR;

`publish`
An aggregate task that publishes all defined publications to all defined repositories. It does not include copying publications to the local Maven cache.

`publishToMavenLocal`
Copies all defined publications to the local Maven cache, including their metadata (POM files, etc.).

So basically you should be able to do` ./gradlew publish -PRELEASE=true -PSNAPSHOT=true` and it should attempt to publish the artifacts to both release and snapshot repos. 

`-PLOCAL` is not needed because `publishToMavenLocal` handles the publish procedure to a local repository